self-core-impl/src/test/resources/responses_en.properties

```diff
@@ -14,4 +14,5 @@ taskAssigned.comment=@%s this is your task now, please go ahead. Deadline (when
                      Estimation here is ``%s minutes``, that's how much you will be paid.\n\n\
                      Remember, you don't have to solve everything in this ticket - you can solve it partially\
                      and leave ``todo`` markers in the code, which will become future tasks.\n\n\
-                     If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.
\ No newline at end of file
+                     If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.
+misunderstand.comment=Didn't understand.
```

Comment:
@criske Can you add ``@%s`` in the beginning here? Like: ``@%s didn't understand that, sorry.``